examples/external-otlp-tonic-tokio/README.md

```diff
@@ -0,0 +1,4 @@
+# Basic OpenTelemetry Example
```

Comment:
Maybe it would be helpful to provide more details about this example here? I think the README.md now is identical with the `basic-otlp` example

Reply:
Hey there, have a look one more time and let me know if it is useful.